src/Sylius/Behat/Context/Ui/Admin/ProductShowPageContext.php

```diff
@@ -60,6 +60,15 @@ public function iAmBrowsingProducts(): void
         $this->indexPage->open();
     }
 
+    /**
+     * @When I try to reach unexistent product
+     */
+    public function iTryToReachUnexistentProductPage(): void
+    {
+        putenv('APP_DEBUG=0');
+        $this->productShowPage->tryToOpen(['id' => 'd', '_locale' => 'en_US']);
```

Comment:
Using random integer value instead of string can probably fix the PostgreSQL build 🤔 
```suggestion
        $this->productShowPage->tryToOpen(['id' => 0, '_locale' => 'en_US']);
```